### PR TITLE
Move golang download to construction time

### PR DESCRIPTION
**Pull request by sttts**
Necessary if the rules file comes from a repo.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,7 +50,7 @@ validate:
 .PHONY: validate
 
 init-deploy: validate
-	$(KUBECTL) delete -n "$(NAMESPACE)" --ignore-not-found=true rs publisher
+	$(KUBECTL) delete -n "$(NAMESPACE)" --ignore-not-found=true replicaset publisher
```
Comment (dims) [inline Makefile]: unrelated. but i am ok with merging it in this PR if you wish

Comment (sttts) [reply Makefile]: rs should even work. It didn't, maybe for some completely unrelated reason.

**Pull request merged**